pkg/fleetautoscalers/controller.go

```diff
@@ -152,6 +150,22 @@ func NewController(
 	return c
 }
 
+// NewExtensions binds the handlers to the webhook outside the initialization of the controller
+// initializes a new logger for extensions.
+func NewExtensions(wh *webhooks.WebHook) *Extensions {
+	ext := &Extensions{
+		extensionLogger: runtime.NewLoggerWithSource("FleetAutscalerExtensions"),
+	}
```

Comment:
```suggestion
	ext := &Extensions{}
  ext.baseLogger = runtime.NewLoggerWithType(ext)
```

To be consistent with how controllers are setup. i.e.

https://github.com/googleforgames/agones/blob/efb2caa77645f1c5d452ccc5e590c13e250bc08c/pkg/gameservers/controller.go#L143

Reply:
Done.